### PR TITLE
Bench 138: Styling Adjusted

### PR DESCRIPTION
**Pull request by SWhyteAnswer**
image
image
image

### CODE REVIEW

**Review by hazel-davis — APPROVED** [review]

**Comment by mbaltrimas** [comment]
image

seems like the mobile navbar is still using the old colour scheme

**Comment by mbaltrimas** [comment]
chrome_hg4DFTjyyN

also the translucent overlay overflows the page

**Comment by mbaltrimas** [comment]
chrome_yv7vpT1q1c

the x button needs slight adjustment as well, need to hover above for it to be clicked - that `float: right` might need to be replaced with something more suitable

File: src/components/OrderPanel/OrderPanel.tsx [hunk 1]
```diff
@@ -1,51 +1,52 @@
...
+  toggleOrderPanel(): void;
+}
+
+export const OrderPanel = ({ toggleOrderPanel }: Props): ReactElement => {
+  const { order, clearOrder } = useOrder();
+
+  const handleClear = (): void => clearOrder();
+
+  return (
+    <div className= "translucent_overlay">
```
Comment (mbaltrimas) [inline src/components/OrderPanel/OrderPanel.tsx]: chrome_hg4DFTjyyN

also the translucent overlay overflows the page

File: src/components/OrderPanel/OrderPanel.tsx [hunk 2]
```diff
@@ -1,51 +1,52 @@
...
+      <div className="order_panel">
+        <div
+          className="order_panel__close"
+          onClick={toggleOrderPanel}
+          onKeyDown={toggleOrderPanel}
+          role="button"
+          tabIndex={0}
+        >
+        &times;
```
Comment (mbaltrimas) [inline src/components/OrderPanel/OrderPanel.tsx]: chrome_yv7vpT1q1c

the x button needs slight adjustment as well, need to hover above for it to be clickable - the `float: right` in the `order_panel__close` class might need to be replaced with something else

**Review by mbaltrimas — CHANGES_REQUESTED** [review]

File: .vs/VSWorkspaceState.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
+  "ExpandedNodes": [
```
Comment (harrystead2) [inline .vs/VSWorkspaceState.json]: What's this?

File: src/components/ItemCreateForm/ItemCreateForm.scss [hunk 4]
```diff
@@ -1,18 +1,25 @@
...
   margin-bottom: 16px;
+  margin-top:8px;
+  border-radius: 7px;
+  border-style: hidden;
+  
 
   &__button {
     width: 100%;
     margin: 8px 0px;
+    border-radius: 7px;
```
Comment (harrystead2) [inline src/components/ItemCreateForm/ItemCreateForm.scss:18]: Set the button border-radius px to a variable in the variables folder

File: src/components/ItemsTable/ItemsTable.scss [hunk 5]
```diff
@@ -2,13 +2,17 @@
 
 .items_table {
   width: 100%;
+  color: $grey;
   border-spacing: 0px;
-  background-color: rgba($grey, 0.8);
+  background-color: whitesmoke;
```
Comment (harrystead2) [inline src/components/ItemsTable/ItemsTable.scss]: create a variable for background-color

File: .vs/VSWorkspaceState.json (hunk 3 above)
Comment (SWhyteAnswer) [reply .vs/VSWorkspaceState.json]: No idea. I'll delete.

File: src/components/OrderPanel/OrderPanel.tsx (hunk 1 above)
Comment (SWhyteAnswer) [reply src/components/OrderPanel/OrderPanel.tsx]: Fixed

File: src/components/OrderPanel/OrderPanel.tsx (hunk 2 above)
Comment (SWhyteAnswer) [reply src/components/OrderPanel/OrderPanel.tsx]: Fixed. Used text allign

File: src/components/ItemsTable/ItemsTable.scss (hunk 5 above)
Comment (SWhyteAnswer) [reply src/components/ItemsTable/ItemsTable.scss]: done

Comment (harrystead2) [reply src/components/ItemsTable/ItemsTable.scss]: use the $form-background variable here

File: README.md [hunk 6]
```diff
@@ -15,4 +15,6 @@ If you are running your API locally, you will need to configure vite to proxy an
 Install extension ESLint. Enter 'npm run lint' to get a error list in the terminal.
 To disable rule on next line comment '// eslint-disable-next-line' followed by the rule name e.g '// eslint-disable-next-line no-alert'.
 See .eslintrc.jc for rules. 
+To fix all lint errors 'npm run lint -- --fix'
```
Comment (harrystead2) [inline README.md:18]: What are the changes to the files above this ReadMe it won't let me select them for some reason? - slnx, suo. read.lock, etc.

File: src/components/ItemCreateForm/ItemCreateForm.scss [hunk 7]
```diff
@@ -1,18 +1,25 @@
...
+
+  background-color: $form-background;
+  color: $grey;
   border: 1px $yellow solid;
   padding: 8px;
   margin-bottom: 16px;
+  margin-top:8px;
+  border-radius: $borderRadius;
+  border-style: hidden;
+
```
Comment (harrystead2) [inline src/components/ItemCreateForm/ItemCreateForm.scss:12]: unnecessary space

File: src/components/NavigationMobile/NavigationMobile.scss [hunk 8]
```diff
@@ -6,12 +6,16 @@
...
+    color: $white;
     font-size: 22px;
     display: flex;
     justify-content: center;
     align-items: center;
-    width: 100%;
+    width: 98%;
+    margin-left: 1%;
+    margin-bottom: 1.5%;
+
```
Comment (harrystead2) [inline src/components/NavigationMobile/NavigationMobile.scss]: 2 unnecessary spaces

File: README.md (hunk 6 above)
Comment (SWhyteAnswer) [reply README.md:18]: + 'To fix all lint errors 'npm run lint -- --fix''

File: src/components/ItemCreateForm/ItemCreateForm.scss (hunk 7 above)
Comment (SWhyteAnswer) [reply src/components/ItemCreateForm/ItemCreateForm.scss:12]: removed

File: src/components/NavigationMobile/NavigationMobile.scss (hunk 8 above)
Comment (SWhyteAnswer) [reply src/components/NavigationMobile/NavigationMobile.scss]: removed

**Comment by harrystead2** [comment]
Remove the changes to the files above the ReadMe

**Review by harrystead2 — APPROVED** [review]

**Review by mbaltrimas — APPROVED** [review]

**Pull request merged**